pkg/plugin/validator.go

```diff
@@ -63,7 +64,7 @@ func NewValidator(ghClinet *github.Client, ghApp *githubapp.GitHubApp) *Validato
 func (v *Validator) MatchIssue(ctx context.Context, issueURL string) error {
 	info, err := parseIssueInfoFromURL(issueURL)
 	if err != nil {
-		return fmt.Errorf("failed to parse issueURL: %w", err)
+		return errors.Join(errInvalidJustification, fmt.Errorf("failed to parse issueURL: %w", err))
```

Comment:
I thought about join errors in `func parseIssueInfoFromURL` instead of here, but I think doing it here make the code simpler.